kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js

```diff
@@ -88,138 +84,141 @@ define([
                 referenceType: 'name',
                 paramsChannelName: bus.channelName,
                 closeParameters: closeParameters,
```

Comment:
another one that could be singularised (i.e. just use `closeParameters`)

it's not worth changing the PR just for this, though.